### PR TITLE
fix: 906 edit avatar style

### PR DESCRIPTION
**Pull request by 310387**
fix for https://github.com/twentyhq/twenty/issues/906

i added fontsize prop to avatar, and at issue for font size it was 16px medium. i used existing medium font which in rem. also i can add in px?

### CODE REVIEW

File: front/src/modules/users/components/Avatar.tsx [hunk 1]
```diff
@@ -13,6 +14,7 @@ type OwnProps = {
   placeholder: string;
```
Comment (charlesBochet) [inline front/src/modules/users/components/Avatar.tsx:14]: let's introduce a size enum (see IconButton for example)
Small | Medium | big

And let's compute height, width and font-size properties from this (see IconButton for example)

**Comment by charlesBochet** [comment]
Thank you! Added a comment!

File: front/src/modules/users/components/Avatar.tsx (hunk 1 above)
Comment (310387) [reply front/src/modules/users/components/Avatar.tsx:14]: what should be the mapping? for iconbutton its

`      case 'large':
        return '32px';
      case 'medium':
        return '24px';
      case 'small':
      default:
        return '20px';`
        
       but for avatar you want 40px. what should be mapping for avatar sizes? and also for font size what should be the mapping? i assume for height and width medium is 40px and for font size 16px. what about small and big sizes?

Comment (charlesBochet) [reply front/src/modules/users/components/Avatar.tsx:14]: Good point!
We have discussed that with our design team, you'll find the different sizes and fontSize in the figma: https://www.figma.com/file/xt8O9mFeLl46C5InWwoMrN/Twenty?node-id=7027%3A100179&mode=dev

So we want 5 sizes: xs, sm, md, lg, xl
Thanks a lot, the help is much appreciated

**Review by charlesBochet — COMMENTED** [review]
Replied in comment above

**Comment by 310387** [comment]
i added the maping and necessary changes.

File: front/src/modules/users/components/Avatar.tsx [hunk 2]
```diff
@@ -6,10 +6,11 @@ import { stringToHslColor } from '~/utils/string-to-hsl';
 import { getImageAbsoluteURIOrBase64 } from '../utils/getProfilePictureAbsoluteURI';
 
 export type AvatarType = 'squared' | 'rounded';
+export type AvatarSize = 'xl' | 'lg' | 'md' | 'sm' | 'xs';
```
Comment (charlesBochet) [inline front/src/modules/users/components/Avatar.tsx:9]: it works but we prefer using enums. Not blocking merged as we still have a few type like this in the codebase

**Review by charlesBochet — APPROVED** [review]
Thank you!

**Comment by charlesBochet** [comment]
Thank you!

**Pull request merged**